applications/asset_tracker/Kconfig

```
@@ -146,6 +146,15 @@ endmenu # GPS
 
 menu "Device and modem"
 
+config CELL_POS_MULTICELL
+	bool "Enable multi cell instead of single cell"
```

Ass per the guidelines, please remove `Enable`:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe just write:
```suggestion
	bool "Multi cell positioning"
```
In the help text you can then describe that this can be used instead on single cell.

---

Spelling: multicell and single cell